Stellar-contract.x

```diff
@@ -59,18 +59,17 @@ enum SCValType
     SCV_VEC = 16,
     SCV_MAP = 17,
 
-    // SCContractExecutable and SCAddressType are types that gets used separately from
-    // SCVal so we do not flatten their structures into separate SCVal cases.
-    SCV_CONTRACT_EXECUTABLE = 18,
-    SCV_ADDRESS = 19,
-
-    // SCV_LEDGER_KEY_CONTRACT_EXECUTABLE and SCV_LEDGER_KEY_NONCE are unique
-    // symbolic SCVals used as the key for ledger entries for a contract's code
-    // and an address' nonce, respectively.
-    SCV_LEDGER_KEY_CONTRACT_EXECUTABLE = 20,
-    SCV_LEDGER_KEY_NONCE = 21,
-
-    SCV_STORAGE_TYPE = 22
+    // The following are the internal SCVal variants that are not
+    // exposed to the contracts. 
+    SCV_ADDRESS = 18,
```

Comment:
Isn't Address exposed?

Reply:
It is, just messed this up during reordering...

Reply:
https://github.com/stellar/stellar-xdr/pull/117